packages/cdk/tsconfig.json

```diff
@@ -1,12 +1,9 @@
 {
-  "extends": "@guardian/tsconfig/tsconfig.json",
-  "compilerOptions": {
-    "module": "CommonJS"
-  },
-  "include": ["lib/**/*", "bin/**/*"],
-  "exclude": [
-    "node_modules",
-    "cdk.out",
-    "lib/**/__snapshots__/**"
-  ]
+	"extends": "@guardian/tsconfig/tsconfig.json",
+	"compilerOptions": {
+		"module": "CommonJS",
+		"noUnusedLocals": false
```

Comment:
ah thanks, no we don't - I just allowed them because it was getting annoying when I was debugging the CDK by commenting stuff out to have to delete all the `const blah =` assignments that were no longer used as a result